{"name": "Проверка логина c неправильным паролем", "status": "passed", "description": "Пытаемся залогиниться в аккаунт курьера, получаем ошибку 404 и соотвествующее сообщение", "start": 1718708392423, "stop": 1718708392548, "uuid": "1cd2693d-efab-49a2-b122-67733328fd49", "historyId": "59c1190257832c75b2535cb60e758ffd", "testCaseId": "59c1190257832c75b2535cb60e758ffd", "fullName": "tests.test_courier_login.TestLoginCourier#test_no_such_username_and_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-02PLGO0"}, {"name": "thread", "value": "17164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}